### ISSUE TITLE
Does it support .tiff images?

### ISSUE THREAD

**Issue by hassanms** [SkalskiP/make-sense#148]
Great project guys,
 
I am trying to annotate some .tiff images but upon uploading the tiff image, the editor area becomes blank and the sidebar image keeps on loading.

How can I run this project on tiff images?

![image](https://user-images.githubusercontent.com/42861376/108601286-ac337a00-73bd-11eb-8600-63ab8a416768.png)

**Comment by SkalskiP** [comment]
Hi @hassanms! We intentionally do not support images in this format due to file size concerns. How large are the files that you want to label?

**Comment by hassanms** [comment]
Hello @SkalskiP. I am glad to hear back from you. The images I want to label are gigantic satellite images but I can crop them to say 500x500 patches. 

How can I add the support for tiff?

**Comment by hassanms** [comment]
@SkalskiP Would you please point me to a way to making the project run tiff images?

**Comment by SkalskiP** [comment]
Hi @hassanms To allow loading of tiff images you would have to modify the application code. Exactly this [file](https://github.com/SkalskiP/make-sense/blob/cd3d26e19a74a58f37cfe2a3a45fbf41e7763177/src/data/enums/AcceptedFileType.ts). But I wonder if it wouldn't be easier for you to just convert the files from tiff to png lib jpg? Since you are using 500x500px crops anyway.

**Comment by hassanms** [comment]
@SkalskiP Yes I modified that file and added the desired format, that's how I was able to upload the tiff images in the first place.

Thanks for your suggestion. Would converting the tiff to png degrade the quality of the image, what do you think?

**Comment by SkalskiP** [comment]
@hassanms It might but I don't think that it would stop you from labeling. The quality loss shouldn't be significant.
